recipes/napari-deeplabcut/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "napari-deeplabcut" %}
+{% set version = "0.0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/napari_deeplabcut-{{ version }}.tar.gz
+  sha256: e14b5022bd85429f042bc08b6d3a2500862a43cac514d1287bd1e30676adff1f
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.8
+    - setuptools-scm
+  run:
+    - dask-image
+    - napari <0.5.0
```

Comment:
This feels unnecessary, but I don't think it hurts either. napari is still on 0.4.x right?